system/Debug/Kint/RichRenderer.php

```diff
@@ -36,11 +36,11 @@ public function preRender()
 
                 switch ($type) {
                     case 'script':
-                        $output .= '<script {csp-script-nonce} class="kint-rich-script">' . $contents . '</script>';
+                        $output .= '<script ' . csp_script_nonce() . ' class="kint-rich-script">' . $contents . '</script>';
```

Comment:
Will these work with View Parser? I don't use it myself but I know people like their Twig-like variable injection...

Reply:
Good point. I added plugins for View Parser.